packages/cody/src/lib/hooks/utils/registry.ts

```diff
@@ -215,6 +215,31 @@ export const registerCommandHandler = (service: string, aggregate: Node, ctx: Co
   return true;
 }
 
+export const registerAggregateRepository = (service: string, aggregate: Node, ctx: Context, tree: FsTree): boolean | CodyResponse => {
+  const serviceNames = names(service);
+  const aggregateNames = names(aggregate.getName());
+
+  if(!isNewFile(
+    joinPathFragments('packages', 'be', 'src', 'repositories', serviceNames.fileName, aggregateNames.fileName, `repository.ts`),
+    tree
+  )
+  ) {
+    return true;
+  }
+
+  const registryPath = joinPathFragments('packages', 'be', 'src', 'repositories', 'index.ts');
+  const registryVarName = 'repositories';
+  const entryId = `${serviceNames.className}.${aggregateNames.className}`;
+  const entryValue = `${serviceNames.propertyName}${aggregateNames.className}`;
+  const importName = `${serviceNames.propertyName}${aggregateNames.className}`;
+  const importPath = `@server/repositories/${serviceNames.fileName}/${aggregateNames.fileName}/repository`;
+
+
+  addRegistryEntry(registryPath, registryVarName, entryId, entryValue, importName, importPath, tree, true);
```

Comment:
This helper function is a wrapper around [ts-morph](https://ts-morph.com/) to add new entries to existing registry files of different types.

Registry files are used across the application to resolve functions and types at runtime which are generated by Cody.